README.md

```diff
@@ -77,3 +77,27 @@ The current NFT contract is deployed on the Mumbai testnet here: [0x35b5147E7499
 
 The current Token contract is deployed on the Mumbai testnet here: [0xc4C99f33F686A74a2Fe95B26Ce317708f605A9eA](https://mumbai.polygonscan.com/address/0xc4C99f33F686A74a2Fe95B26Ce317708f605A9eA)
 
+## Setup CI
+### Setup Slither-Analyzer
+
+Slither is a Solidity static analysis framework written in Python 3. It runs to analyze code vulnerabilities according to security aspects.
+
+Environment
+```
+python 3.6+
+```
+
+Setup dependencies:
+```
+sudo snap install solc
+python -m pip install --upgrade pip
+pip install slither-analyzer solc-select
```

Comment:
We should create a `requirements.txt` file for this and add instructions how to setup a `venv`. SOmething like this should be enough.

Create a new virtual environment:
```
python -m venv .venv
source .venv/bin/activate
```

Install the required Python packages:
```
pip install -r requiremets.txt
```